webviz_config/_plugin_abc.py

```diff
@@ -177,7 +202,24 @@ def plugin_layout(
         if self._add_download_button:
             buttons.append("download")
 
-        if buttons:
+        extended_deprecation_warnings: List[Dict[str, str]] = []
+
+        if deprecation_warnings:
+            name = getattr(self, "__module__").lower()
+            name_parts = name.split(".")
+            distribution_name = name_parts[0]
```

Comment:
It would be nice if we found a way to use this instead of defining a new way to get the information. However, I do always end up with circular import errors when trying to import from `plugins._utils`.